tests/modules/sightings/resources/test_sighting_assets.py

```diff
@@ -70,15 +70,14 @@ def test_asset_addition(db, flask_app_client, staff_user):
         new_asset_1.delete()
         new_asset_2.delete()
         new_asset_3.delete()
+        with db.session.begin():
+            db.session.delete(new_asset_group)
```

Comment:
why not new_asset_group.delete()

Reply:
Same fk issue with sighting_assets. Perhaps deserves a bug ticket of it's own.

Reply:
Had to use:
`new_asset_group.delete_remote()
 asset_group_utils.delete_asset_group(
        flask_app_client, researcher_1, new_asset_group.guid
 )`
after rebasing.